install.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 # Assumes the script is called from PYLOT_HOME directory
+###### Build the FrenetOptimalTrajectory Planner ######
+PYLOT_HOME=$(pwd)
+sudo apt-get install -y libeigen3-dev
```

Comment:
Do you want add these dependencies in your repository in an `install.sh` or `build.sh` script. I think apt-get install should also get clang and cmake. Thus, we can keep the pylot install minimal.

Reply:
Sounds good, done!